tests/plugins/uploadimage/manual/afterreplaceselection.md

```diff
@@ -0,0 +1,19 @@
+@bender-tags: 4.9.0, bug, 1217
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, undo, uploadwidget, basicstyles, image2, uploadimage, uploadfile, font, stylescombo, basicstyles, format, maximize, blockquote, list, table, resize, elementspath, justify
+@bender-include: ../../uploadwidget/manual/_helpers/xhr.js
+
+1. Copy several custom images.
```

Comment:
AFAIR this behaviour is present only in Firefox, so this test is not working anywhere else.